tools/hidl/hidlcomp.cpp

```diff
@@ -3504,224 +2900,12 @@ void EspMessageInfo::write_esp()
         }
     }
 
-    if (typesNeeded.size()>0)
-    {
-        for (TypeMap::const_iterator it = typesNeeded.begin(); it!=typesNeeded.end(); ++it)
-        {
-            outf("\tif (!added.getValue(\"%s\"))\n", (*it).first.c_str());
-            outf("\t{\n");
-            outf("\t\tadded.setValue(\"%s\",1);\n", (*it).first.c_str());
-            outf("\t\tC%s%s::getMapInfo(info,added);\n",((*it).second & MIT_IsEnum)?"X":"",(*it).first.c_str());
-            outf("\t}\n");
-        }
-    }
-
     outs("}\n");
 
     //=======================================================================================
```

Comment:
I think this  three line comment block is for the getHtmlForm function output which was removed, so the comment should be too.

Reply:
Removed.